components/user_settings/notifications/notification_schedule_setting.scss

```diff
@@ -0,0 +1,194 @@
+.form-switch {
+    display: flex;
+    align-items: center;
+    border: none;
+    margin: 0 0 0.8rem;
+    color: inherit;
+    font-size: inherit;
+    font-weight: 600;
+
+    .switch {
+        position: relative;
+        display: inline-block;
+        width: 48px;
+        height: 28px;
+        margin-right: 8px;
+
+        input {
+            width: 0;
+            height: 0;
+            opacity: 0;
+        }
+
+        .slider {
+            position: absolute;
+            top: 0;
+            right: 0;
+            bottom: 0;
+            left: 0;
+            background-color: #ccc;
+            cursor: pointer;
+            -webkit-transition: 0.4s;
+            transition: 0.4s;
+        }
+
+        .slider::before {
+            position: absolute;
+            bottom: 4px;
+            left: 4px;
+            width: 20px;
+            height: 20px;
+            background-color: white;
+            content: "";
+            -webkit-transition: 0.4s;
+            transition: 0.4s;
+        }
+
+        input:checked + .slider {
+            background-color: #1b70ff;
```

Comment:
please use `background-color: var(--button-bg)` to ensure this uses the appropriate theme color